src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +16,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        emailUserFound.findByEmail(email);
+
+        if (emailUserFound.findByEmail(email) != null) {
+            User posUser = emailUserFound.findByEmail(email);
```

Comment:
call findByEmail method only one time
Think about var names: **posUser**, **mbPass** - I can't understand what is it